modules/virt-about-services.adoc

```diff
@@ -8,7 +8,7 @@
 
 A Kubernetes _service_ exposes network access for clients to an application running on a set of pods. Services offer abstraction, load balancing, and, in the case of NodePort and LoadBalancer, exposure to the outside world. 
 
-Services can be exposed in different ways by specifying a `spec.type` in the `Service` object:
+Services can be exposed in the *Details* tab of the web console, or by specifying a `spec.type` in the `Service` object:
```

Comment:
I'm not doing a peer review. This just caught my eye. Please remove the comma. It is not required for "x or y" construction. Thanks.